File: Principles of Programming Language/HW8P/TriangleIf.py
#Author: Matt McCullough

a = int(input("Enter an integer: "));
b = int(input("Enter another integer: "));
c = int(input("Enter another integer: "));

flag = 1;
bool(flag);

if((a+b)<=c):
    flag = 0;
if((b+c)<=a):
    flag = 0;
if((c+a)<=b):
    flag = 0
if(flag == 1):
    print("Valid triangle");
else:
    print("Invalid triangle");

if((a+b)>c):
    if((b+c)>a):
        if((c+a)>b):
            print("Valid triangle");
else:
    print("Invalid triangle");

if((a+b)>c and (b+c)>a and (c+a)>b):
    print("Valid triangle");
else:
    print("Invalid triangle");

File: Principles of Programming Language/HW8J/TriangleIf.java
import java.util.Scanner;

//Author: Matt McCullough
public class TriangleIf {
    public static void main(String[] args)
    {
        Scanner reader = new Scanner(System.in);
        System.out.print("Enter an integer: ");
        int a = reader.nextInt();
        System.out.print("Enter another integer: ");
        int b = reader.nextInt();
        System.out.print("Enter another integer: ");
        int c = reader.nextInt();

        boolean flag = true;

        if((a+b)<=c)
            flag = false;
        if((b+c)<=a)
            flag = false;
        if((c+a)<=b)
            flag = false;
        if(flag == true)
        {
            System.out.println("Triangle is valid");
        }
        else
            System.out.println("Invalid triangle");

        if((a+b)>c)
        {
            if((b+c)>a){
                if((c+a)>b)
                    System.out.println("Triangle is valid");
            }
        }
        else
            System.out.println("Invalid triangle");

        if((a+b)>c && (b+c)>a && (c+a)>b)
        {
            System.out.println("Triangle is valid");
        }
        else
            System.out.println("Invalid triangle");



    }

}
